src/query/test/testdata/aggregators.test

```diff
@@ -0,0 +1,304 @@
+load 5m
```

Comment:
May need to add a `LICENSE.txt` in this folder with the uber + prometheus header